apps/projection/src/app/ui/card/card.component.ts

```diff
@@ -1,34 +1,44 @@
 import { NgFor, NgIf } from '@angular/common';
-import { Component, Input } from '@angular/core';
-import { randStudent, randTeacher } from '../../data-access/fake-http.service';
+import {
+  Component,
+  Input,
+  TemplateRef,
+  Output,
+  EventEmitter,
+} from '@angular/core';
 import { StudentStore } from '../../data-access/student.store';
 import { TeacherStore } from '../../data-access/teacher.store';
+import { CityStore } from '../../data-access/city.store';
 import { CardType } from '../../model/card.model';
 import { ListItemComponent } from '../list-item/list-item.component';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'app-card',
   templateUrl: './card.component.html',
   standalone: true,
-  imports: [NgIf, NgFor, ListItemComponent],
+  imports: [NgIf, NgFor, ListItemComponent, CommonModule],
+  //providers: [TemplateRef],
 })
 export class CardComponent {
   @Input() list: any[] | null = null;
   @Input() type!: CardType;
   @Input() customClass = '';
+  @Input() templateView: TemplateRef<any>;
```

Comment:
template should almost never be pass as input. 

You can get them using `@ContentChild`

Reply:
done